ci/envs/37-minimal.yaml

```diff
@@ -19,7 +19,7 @@ dependencies:
   - rtree
   - matplotlib
   - matplotlib=3.1
-  - mapclassify>=2.2.0
+  - mapclassify>=2.4.0
```

Comment:
```suggestion
  - mapclassify=2.4.0
```

would this work as well? (since in this build we try to test against minimum versions we support)

Reply:
2.4.0 is the latest so there should be no difference at the moment.